### ISSUE TITLE
python3 Errno13

### ISSUE THREAD

**Issue by PicoRadia** [scrapinghub/splash#1036]
when i run :

sudo docker run -p 8000:8080 scrapinghub/splash 

I get:
python3: can't open file '/app/bin/splash': [Errno 13] Permission denied

Any solution please?

**Comment by lucywang000** [comment]
Have you using a stable release like `scrapinghub/splash:3.4.1`?

**Comment by PicoRadia** [comment]
yes i get this same error and i just can't get over it  or find a solution . Even if i change the permissions for '/app/bin/splash' as sudo it's the same error . I'm running Docker version 18.09.9, build 1752eb3
 on Linux Mint
